.storybook/stories.css

```diff
@@ -0,0 +1,14 @@
+.story-component {
```

Comment:
This looks cumbersome, did you write this style yourself? Is there a before after?

Reply:
Added a comment. This adds the blueprint background behind components.